x-pack/plugins/cases/public/components/user_actions/pushed.tsx

```diff
@@ -112,31 +109,25 @@ const getFooters = ({
 export const createPushedUserActionBuilder: UserActionBuilder = ({
   userAction,
   userProfiles,
-  caseServices,
-  index,
+  caseConnectors,
   handleOutlineComment,
 }) => ({
   build: () => {
     const pushedUserAction = userAction as UserActionResponse<PushedUserAction>;
-    const { firstPush, parsedConnectorId, parsedConnectorName } = getPushInfo(
-      caseServices,
-      pushedUserAction.payload.externalService,
-      index
-    );
+    const connectorId = pushedUserAction.payload.externalService?.connectorId;
```

Comment:
Is it possible that the `externalService` can be undefined? I think the checks below are fine to be extra safe but I think `connectorId` will always be a string.

Reply:
I think you are right. `externalService` should be always defined. About the `connectorId` a bit paranoid but I wanted to guarded for this scenario https://github.com/elastic/kibana/blob/b5b626bc160bce6b029de97efbe7a66aaecde30c/x-pack/plugins/cases/server/saved_object_types/migrations/user_actions/payload.ts#L171